pt_dgt/pt_dgt_README.txt

```diff
@@ -9,12 +9,12 @@ from [DG Territorio](http://www.dgterritorio.pt/)
 
 Grid transformation from Portuguese Lisbon Datum to ETRS89 in Portugal.
 
-*Source*: [DGT](http://www.dgterritorio.pt/cartografia_e_geodesia/geodesia/transformacao_de_coordenadas/grelhas_em_ntv2/)  
-*Format*: GeoTIFF converted from NTv2  
-*License*: [Creative Commons Attribution 4.0](http://creativecommons.org/licenses/by/4.0/)  
+*Source*: [DGT](http://www.dgterritorio.pt/cartografia_e_geodesia/geodesia/transformacao_de_coordenadas/grelhas_em_ntv2/)
```

Comment:
the 2 trailing spaces are intended for correct formatting in github

Reply:
restored